nasl/nasl_grammar.y

```diff
@@ -460,8 +461,8 @@ expr: '(' expr ')' { $$ = $2; }
 	| post_pre_incr
 	| expr MATCH expr { $$ = alloc_expr_cell(LNB, COMP_MATCH, $1, $3); }
 	| expr NOMATCH expr { $$ = alloc_expr_cell(LNB, COMP_NOMATCH, $1, $3); }
-	| expr RE_MATCH string { $$ = alloc_RE_cell(LNB, COMP_RE_MATCH, $1, $3); }
-	| expr RE_NOMATCH string { $$ = alloc_RE_cell(LNB, COMP_RE_NOMATCH, $1, $3); }
+        | expr RE_MATCH string { $$ = alloc_RE_cell(LNB, COMP_RE_MATCH, $1, $3, ERRC); }
+        | expr RE_NOMATCH string { $$ = alloc_RE_cell(LNB, COMP_RE_NOMATCH, $1, $3, ERRC); }
```

Comment:
```suggestion
    | expr RE_MATCH string { $$ = alloc_RE_cell(LNB, COMP_RE_MATCH, $1, $3, ERRC); }
    | expr RE_NOMATCH string { $$ = alloc_RE_cell(LNB, COMP_RE_NOMATCH, $1, $3, ERRC); }
```

Reply:
I don't want to modify the format of the nasl_grammar.y. It was written with tabs instead of spaces  and longer lines than 80 chars everywhere. It doesn't compliance with the current clang format we uses for .c files
It would deserve an own PR.